dbms/src/Storages/Page/V3/CheckpointFile/CPFilesWriter.cpp

```diff
@@ -100,6 +100,11 @@ CPDataDumpStats CPFilesWriter::writeEditsAndApplyCheckpointInfo(
         StorageType id_storage_type = StorageType::Unknown;
         {
             id_storage_type = UniversalPageIdFormat::getUniversalPageIdType(rec_edit.page_id);
+            if (id_storage_type == StorageType::LocalKV)
+            {
+                // These pages only contains local data which will not be dumped into checkpoint.
+                continue;
+            }
```

Comment:
should put these lines after updating the `write_down_stats`

Reply:
Why? These keys are written into neither manifest file nor data file

Reply:
`stats.num_existing_bytes` is reported to `GET_METRIC(tiflash_storage_page_data_by_types, type_localkv).Set(stats.num_existing_bytes[i])`
which represent how many bytes of this type are stored locally

Reply:
fixed